### DISCUSSION TITLE
How to add a todo with priority, without going to  edit.

### DISCUSSION THREAD

**Discussion by debajyoti1990** [VladimirMarkelov/ttdl#80] [General]
I couldn't find any example which shows addition of a todo with a certain priority.

 I mean while creation of a todo we can specify project, context, due and recurring. 

e.g. ttdl a "send tax declaration +personal @finance @tax due:2018-04-01 rec:1y"

For adding priority we don't have any other option than to edit the todo with flag --set-pri: A/B right ?

**Comment by VladimirMarkelov** [comment]
Priority is a part of the task description, it must go first in round brackets:

```
ttdl add "(A) do it immediately"
```
